src/coreclr/vm/interoputil.cpp

```diff
@@ -140,9 +140,9 @@ HRESULT SetupErrorInfo(OBJECTREF pThrownObject)
                     {
                         GCX_PREEMP();
                         if (!message.IsEmpty())
-                            EEMessageBoxNonLocalizedDebugOnly((LPWSTR)message.GetUnicode(), W(".NET exception in COM"), MB_ICONSTOP | MB_OK);
+                            EEMessageBoxNonLocalizedNonFatal((LPWSTR)message.GetUnicode(), W(".NET exception in COM"), MB_ICONSTOP | MB_OK);
```

Comment:
This can be changed to asserts or debug breaks.

Reply:
I think so too, but I didn't want to clean up that much without someone making a comment. This means I am going to remove some COM config flags.